ledger-api/grpc-definitions/com/daml/ledger/api/v2/commands.proto

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v2;
+
+import "com/daml/ledger/api/v1/commands.proto";
+
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+
+
+option java_outer_classname = "CommandsOuterClass";
+option java_package = "com.daml.ledger.api.v2";
+option csharp_namespace = "Com.Daml.Ledger.Api.V2";
+
```

Comment:
At least for a while, I would consider documenting the changes between v1 and v2: it will help both review and migration. WDYT?

Reply:
Changelog is good!